kafka/admin/client.py

```diff
@@ -186,6 +188,7 @@ class KafkaAdminClient(object):
         'metric_reporters': [],
         'metrics_num_samples': 2,
         'metrics_sample_window_ms': 30000,
+        'client_factory': None,
```

Comment:
Could we simplify by making this default to `kafka.client_async.KafkaClient` ?

I also think we could probably just call this `'client'` . Note that we don't use 'factory' to describe similar configs like 'selector' and 'metrics_reporters'

Reply:
updated